includes/class-settings-fields.php

```diff
@@ -0,0 +1,384 @@
+<?php
```

Comment:
should we put these under the admin namespace and in an admin subfolder? like `Activitypub\Admin\Settings_Fields`??

Reply:
Yes! There are a few other classes that would fit in there, and I'd like to combine it with hooking them into `admin_init` rather than `init`, so I thought that could be a good follow-up PR